files/es/web/api/screen/index.md

```diff
@@ -0,0 +1,78 @@
+translation_of: Web/API/Screen
+browser-compat: api.Screen
+---
+{{APIRef("CSSOM")}}
+
+La interfaz `Screen` representa una pantalla,
+usualmente la misma donde la ventana actual está siendo visualizada y
+es obtenida usando {{domxref("window.screen")}}.
+
+Tenga en cuenta que los navegadores determinan cual es la pantalla actual
+detectando qué pantalla tiene el navegador centrada.
+
+## Propiedades
+
+- {{DOMxRef("Screen.availTop")}} {{Non-standard_Inline}}
+  - : Especifica la coordenada-y del primer píxel que no está asignado a funciones de interfaz de usuario permanentes o semipermanentes.
+- {{DOMxRef("Screen.availLeft")}} {{Non-standard_Inline}}
+  - : Devuelve el primer píxel disponible desde el border izquierdo de la pantalla.
+- {{DOMxRef("Screen.availHeight")}}
+  - : Especifica la altura de la pantalla, en píxeles, menos las funciones de interfaz de usuario permanentes o semipermanentes que son mostradas por el sistema operativo, como la barra de tareas de Windows.
+- {{DOMxRef("Screen.availWidth")}}
+  - : Devuelve la cantidad de espacio horizontal disponible en pixeles para la ventana.
+- {{DOMxRef("Screen.colorDepth")}}
+  - : Devuelve la profundidad de color de la pantalla.
+- {{DOMxRef("Screen.height")}}
+  - : Devuelve la altura de la pantalla en píxeles.
+- {{DOMxRef("Screen.left")}} {{Non-standard_Inline}}
+  - : Devuelve la distancia en píxeles desde el borde izquierdo de la pantalla principal al borde izquierdo de la pantalla actual.
+- {{DOMxRef("Screen.orientation")}}
+  - : Devuelve la instancia {{domxref("ScreenOrientation")}} asociada con la pantalla.
+- {{DOMxRef("Screen.pixelDepth")}}
+  - : Obtiene la profundidad de bits de la pantalla.
+- {{DOMxRef("Screen.top")}} {{deprecated_inline}}{{Non-standard_Inline}}
+  - : Devuelve la distancia en píxeles desde el borde supierior de la pantalla principal.
+- {{DOMxRef("Screen.width")}}
+  - : Devuelve la anchura de la pantalla.
+- {{DOMxRef("Screen.mozEnabled")}} {{Non-standard_Inline}} {{Deprecated_Inline}}
+  - : Booleano. Estableciendo su valor a `false`, se apagaá la pantalla del dispositivo.
+- {{DOMxRef("Screen.mozBrightness")}} {{Non-standard_Inline}} {{Deprecated_Inline}}
+  - : Controla el brillo de la pantalla del dispositivo. Espera un valor `double` entre `0` y `1.0`.
+
+### Manejador de eventos
+
+- {{DOMxRef("Screen.onorientationchange")}} {{Deprecated_Inline}}
+  - : Un manejador para el evento {{event("orientationchange")}}.
+
+## Métodos
+
+- {{DOMxRef("Screen.lockOrientation")}} {{Non-standard_Inline}} {{Deprecated_Inline}}
+  - : Bloquea la orientación de la pantalla (sólo funciona en pantalla completa o para aplicaciones instaladas)
+- {{DOMxRef("Screen.unlockOrientation")}} {{Non-standard_Inline}} {{Deprecated_Inline}}
+  - : Desbloquea la orientación de pantalla (sólo funciona en pantalla completa o para aplicaciones instaladas)
+
+_Métodos heredados de {{domxref("EventTarget")}}:_
+
+{{Page("/es/docs/Web/API/EventTarget", "Methods")}}
```

Comment:
Aún sigue esta macro en la versión en Inglés y esta funcionando...

@mdn/yari-content-es, que se hace en esos casos?

https://github.com/mdn/content/blob/main/files/en-us/web/api/screen/index.md

Reply:
@cw118 @SphinxKnight sorry for pinging you but we don't get how to handle this.

I've seen the [English version](https://github.com/mdn/content/blob/main/files/en-us/web/api/screen/index.md) is still using the `page` macro.

We're wondering if should we remove it or what should we do in this case? Do you know?

Reply:
All good (and sorry for the late reply), I think it'd be best to wait for SphinxKnight's response but maybe this page macro could be replaced with the [methods found here](https://developer.mozilla.org/es/docs/Web/API/EventTarget#methods) (the heading that this `{{Page}}` transcludes), and a PR can opened at mdn/content to remove the `{{page}}` there?

Reply:
we thought that! just remove the `{{page}}` macro and use the methods there, okay, sounds good, let's wait for SphinxKnight's response, thank you for your help :pray:

Reply:
I think that the replacement of the macro should be a link to the page that it makes reference, instead of duplicating the content.

Reply:
> In this case it works because [the target page of the macro has an explicit `name` attribute](https://github.com/mdn/translated-content/blob/main/files/es/web/api/eventtarget/index.html#L16) for the corresponding section. Keeping this macro call as is would fail if the `EventTarget` page would be updated with `Métodos` (where, in this case `{{Page("/es/docs/Web/API/EventTarget", "Métodos")}}` would work.
> 
> 
> 
> Also in this case, I concur with @davbrito and @cw118, duplicating the content might not be the best call, here should EventTarget be updated (also I assume it's being used in a lot of pages). **Adding a link would be quite a simple solution which would fix the coupling issue (between source and target page) from the `{{page}}` call.**
> 
> 
> 
> To elaborate a bit more, as a rule of thumb, English content is the reference, so as long as the translated page does not have any issue for the reader, sticking to what mdn/content has (in terms of macro calls for instance) is a right "default strategy" (in other words, in this case, if I were to review this PR, I wouldn't block it because it uses the macro since it works and since it is present in the English page).

Thank you for your guidance and justify suggestion! 🙏

Reply:
Nota de referencia, se queda como está funcionando en inglés y se esperará a que se actualice ahí